docs/design/task-DAG-scheduling.md

```diff
@@ -0,0 +1,92 @@
+# Task-level DAG scheduling policy
+
+@[hwdef](https://github.com/hwdef); July 19th, 2021
+
+## Introduction
+
+This feature provides the ability to customize the order in which tasks are launched.
+
+## Scenarios
+
+* mpi job. the master needs to wait for the worker to start before starting, If the master is already started, but the worker is not yet started, the master will restart, which will add unnecessary waste of resources ,in this case, mpiworker needs to be in the running state, and then create the pod for mpimaster..
+
+* Suppose there are two tasks and task 2 needs to use the calculation results of task 1,in this case, task1 needs to be in the complete state, and then create the pod for task2.
+
+* some ETL applications
```

Comment:
Can you give more details about ETL applcations?

Reply:
https://en.wikipedia.org/wiki/Extract,_transform,_load

https://baike.baidu.com/item/ETL/1251949?fr=aladdin

Here are some links about ETL. Simply put, ETL represents the three steps of processing data, and there is a dependency between these three steps, so we need a specific order to start the task.